README.md

```diff
@@ -40,7 +40,7 @@ volume. Customizable parameters for volume creation include:
 | Parameter         | Values                  | Default                                | Description |
 | ---------------   | ----------------------- |-----------                             | ----------- |
 | tier              | "standard"<br>"premium" | "standard"                             | storage performance tier |
-| network           | string                  | "default"                              | VPC name |
+| network           | string                  | Cluster network auto-detected by the driver controller | VPC name |
```

Comment:
Even if we can auto detect the network, the user needs to still create firewall rules right?